app/models/server/raw/Mentions.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseRaw, IndexSpecification } from './BaseRaw';
+import { IMention } from '../../../../definition/IMention';
+
+export class MentionsRaw extends BaseRaw<IMention> {
+	protected indexes: IndexSpecification[] = [
+		{ key: { uid: 1 }, unique: true },
```

Comment:
this index is not needed and will actually prevent to mentions to the same user to be inserted.